app/assets/stylesheets/application.scss

```diff
@@ -105,3 +136,62 @@ body {
   width: 250px;
   padding: 5px;
 }
+
+Masonary
+* {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+body { font-family: sans-serif; }
+
+/* ---- grid ---- */
+
+.grid {
+  background: $mediumaquamarine;
+  max-width: 2000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+/* clearfix */
+.grid:after {
+  content: '';
```

Comment:
Properties should be ordered clear, content, display<br>Prefer double-quoted strings